### PR TITLE
test drupal 10

### PR DESCRIPTION
**Pull request by byrond**
### Description

_A description of the changes. Consider including images/gifs to demonstrate behaviors (e.g. using [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture))._

### Testing instructions

1. `composer create-project palantirnet/drupal-skeleton drupal10 dev-drupal-10 --no-interaction`
2. `cd drupal10`
3. Edit `.ddev/config.yaml` and change the `name` to `drupal10`
4. `ddev start`
5. `ddev ssh`
6. `vendor/bin/the-build-installer`
7. Visit https://drupal10.ddev.site/ and verify the site is running.
8. Log in, and verify the site is running Drupal 10.

### Remaining tasks

- [x] Review and merge https://github.com/palantirnet/the-build/pull/201
- [x] Create a new release of `the-build`
- [x] Change line 30 here: https://github.com/palantirnet/drupal-skeleton/pull/126/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R30 to use `^4@beta` (or `^4` if there is a stable release).

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,19 +15,20 @@
...
-        "drupal/admin_toolbar": "^3.0",
-        "drupal/config_split": "^1.7",
-        "drupal/core-composer-scaffold": "^9",
-        "drupal/core-recommended": "^9",
-        "drupal/devel": ">=4.1",
-        "drupal/workbench": "^1.3",
-        "drupal/workbench_tabs": "^1.5"
+        "composer/installers": "^2.0",
+        "drupal/admin_toolbar": "^3",
+        "drupal/config_split": "^1",
```
Comment (becw) [inline composer.json:20]: I had to use config_split ^2 in another Drupal 10 project I set up. Not sure if we'll need to do that here as well.

Comment (byrond) [reply composer.json:20]: 1.9 is compatible with Drupal 10, and 2.0.0 is still in pre-release, so I think we can leave this for now.

**Comment by Kbentham** [comment]
I am seeing some errors that I think are related to this issue on the module and status pages https://www.drupal.org/project/drupal/issues/3293463

https://drupal10.ddev.site/admin/modules
Screen Shot 2023-01-11 at 1 47 00 PM

https://drupal10.ddev.site/admin/reports/status
Screen Shot 2023-01-11 at 1 47 24 PM
 other than that this is looking good!

**Comment by byrond** [comment]
@Kbentham I added the patch in the Drupal core issue you found, but I still got the same error after first install and going to the status page. Clearing cache made the error go away. I removed the patch, so could you confirm that clearing cache after installing the site allows the status page to work?

**Comment by byrond** [comment]
We might need to update The Build to clear cache when it's done switching the profile, so this manual step isn't required after running the first Drupal install.

**Review by Kbentham — APPROVED** [review]
Looks good to me

**Pull request merged**